exercises/luhn/example.rs

```diff
@@ -0,0 +1,13 @@
+pub fn is_valid(candidate: &str) -> bool {
+    if candidate.chars().any(|c| c.is_alphabetic()) || candidate.chars().count() == 1 {
+        return false;
+    }
+
+    candidate.chars()
+        .filter_map(|c| c.to_digit(10))
+        .rev()
+        .enumerate()
+        .map(|(index, digit)| if index % 2 == 0 { digit } else { digit * 2 })
+        .map(|digit| if digit > 10 { digit - 9 } else { digit })
```

Comment:
sorry, but this code says `!is_valid("59")` and `is_valid("50")`, when instead it should say `is_valid("59")` and `!is_valid("50")`.